.gitignore

```diff
@@ -17,3 +17,5 @@ cmake-build-*/
 .vscode/
 build/
 
+build-out/
+cmake-out/
```

Comment:
Should these belong in the "Common build output directory names" section?

Reply:
Fixed, thanks.